tensorflow/core/common_runtime/mkl_cpu_allocator.h

```diff
@@ -129,8 +132,8 @@ class MklSmallSizeAllocator : public VisitableAllocator {
   }
 
  private:
-  /// Increment statistics for the allocator handling small allocations.
-  inline void IncrementStats(size_t alloc_size) {
+  // Increment statistics for the allocator handling small allocations.
+  inline void IncrementStats(size_t alloc_size) GUARDED_BY(mutex_) {
```

Comment:
`GUARDED_BY` is used on variables. Please use `EXCLUSIVE_LOCKS_REQUIRED(mutex_)` instead. (See [ThreadSafetyAnalysis](http://releases.llvm.org/3.5.1/tools/clang/docs/ThreadSafetyAnalysis.html) for example.)